config/application.rb

```diff
@@ -42,8 +42,8 @@ class Application < Rails::Application
 
     # Caching
     config.cache_control_directive = 'public'
-    config.default_ttl = 30.minutes
-    config.minimum_ttl = 5.seconds
+    config.default_ttl = ENV.fetch('DEFAULT_TTL', '1800').to_i
```

Comment:
I think this could be: `ENV.fetch('DEFAULT_TTL', 30.minutes).to_i` and it would be more readable and comparable with the line below.